src/Dds/Interfaces/ICrudService.cs

```diff
@@ -7,7 +7,7 @@ public interface ICrudService
     {
         StringResponse Create(string storeName, Dictionary<string, string> values);
         StringResponse Delete(string storeName, string identityId);
-        ReadResponse Read(string storeName, int start, int pageSize, string search, int sortByColumn, string sortDirection);
+        ReadResponse Read(string storeName, int start, int pageSize, string search, int sortByColumn, string sortDirection, string filterColumnName, string filter);
```

Comment:
Different naming convention (compared to `sortBy`). Should be `filterByColumn` and `filterValue`.